docs/src/modules/components/DemoSandbox.js

```diff
@@ -172,7 +172,7 @@ function getTheme(outerTheme, injectTheme) {
   if (injectTheme && Object.prototype.toString.call(injectTheme) === '[object Object]') {
     try {
       return deepmerge(resultTheme, injectTheme);
-    } catch (e) {
+    } catch (event) {
```

Comment:
There are few other instances of these

```suggestion
    } catch {
```